### PR TITLE
37 update theme with the one from the style guide

### LINKED ISSUE #37: Update theme with the one from the style guide
- [ ] Remove extras from the tailwind config and app.css

### PR DESCRIPTION
**Pull request by claireolmstead**
# Description

Update theme to use the styleguide.

- Closes #37 

## Type of change

- [x] New Feature

# How to Test?

Pull branch and see that styles look the same and are not broken in any screen size.

1. Pull and run locally
2. See that css is coming through as well as the theme. Best examples are the underline on hover in the nav (showing css) and the colors properly coming through (tailwind config)

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code & PR, particularly in hard-to-understand areas
- [x] I have checked at all the breakpoints to make sure it works on all screen sizes

### CODE REVIEW

File: src/routes/+layout.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import '../style/app.css';
-
+import * as styles from '@frequency-chain/style-guide';
+console.log(styles);
```
Comment (enddynayn) [inline src/routes/+layout.ts:3]: It looks like this might be some leftover from debugging?

File: .idea/.gitignore
Comment (claireolmstead) [inline .idea/.gitignore:1]: remove .idea files and add to gitignore

File: src/components/FounderProfile.svelte [hunk 2]
```diff
@@ -5,7 +5,7 @@
   export let linkedinUrl: string;
 </script>
 
-<div class="max-w-[100px] sm:text-indigo lg:text-cream">
+<div class="max-w-[100px] sm:text-brightBlue lg:text-cream">
```
Comment (claireolmstead) [inline src/components/FounderProfile.svelte:8]: same hex, new name

File: src/components/JoinSocials.svelte [hunk 3]
```diff
@@ -5,7 +5,7 @@
 </script>
 
 <div class="w-full max-w-[425px] pb-[80px] sm:pt-[80px] lg:pl-[140px] lg:pt-6">
-  <h3 class="text-base font-semibold">Join the Digital Revolution</h3>
+  <h3 class="text-normal font-semibold">Join the Digital Revolution</h3>
```
Comment (claireolmstead) [inline src/components/JoinSocials.svelte:8]: new name

**Review by enddynayn — APPROVED** [review]
👍 looks great!

**Pull request merged**